packages/eslint-config-sheriff/tsup.config.json

```diff
@@ -4,5 +4,12 @@
   "dts": true,
   "splitting": false,
   "sourcemap": false,
-  "clean": true
+  "clean": true,
+  "target": "node20",
+  "skipNodeModulesBundle": true,
+  "noExternal": ["@sherifforg/constants"],
```

Comment:
This is great, but in tandem we should also revert the old fix that i did, meaning bringign back this dep to the regular deps instead of devdeps